docs/docusaurus/docs/cloud/access/manage_access.md

```diff
@@ -0,0 +1,175 @@
+3. Review the warning and enter the workspace name to confirm you understand the consequences and want to proceed.
+4. Click **Delete**.
+
+## Users
+
+Workspace users can be members of multiple workspaces with different permissions. Organization Owners are always members of all workspaces with full permissions.
+
+### Roles and permissions
+
+
+The following table lists GX Cloud roles and permissions.
+
+| User role                                            | Organization Owner                          | Workspace Admin                             | Workspace Editor                           | Workspace Viewer                           |
+|------------------------------------------------------|---------------------------------------------|---------------------------------------------|--------------------------------------------|--------------------------------------------|
+| Manage workspaces                                    | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="No">❌ </span> | <span role="img" aria-label="No">❌ </span> | <span role="img" aria-label="No">❌ </span> |
+| Manage Organization Owners                           | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="No">❌ </span> | <span role="img" aria-label="No">❌ </span> | <span role="img" aria-label="No">❌ </span> |
+| Manage organization access tokens                    | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="No">❌ </span> | <span role="img" aria-label="No">❌ </span> | <span role="img" aria-label="No">❌ </span> |
+| Manage workspace users *                             | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="No">❌ </span> | <span role="img" aria-label="No">❌ </span> |
+| Manage user access tokens *                          | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="No">❌ </span> |
+| Manage Data Sources, Data Assets, and Expectations * | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="No">❌ </span> |
+| View Validation Results *                            | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="Yes">✅</span> | <span role="img" aria-label="Yes">✅</span> |
+
+\* Scoped to the workspace(s) the user belongs to.
+
+### Invite a user
+
+1. In GX Cloud, click **Users**.
+
+2. Click **Invite user** and complete the following fields:
+
+    - **Workspace** - If you’re adding a Workspace Admin, Editor, or Viewer, select a workspace. If you’re adding an Organization Owner, they will automatically have full access to all current and future workspaces. 
```

Comment:
Owners will also have access to all existing workspaces as well, not sure if we want to mention it

Reply:
I believe this is already mentioned here:

> If you’re adding an Organization Owner, they will automatically have full access to all current and future workspaces. 

Is there a difference between "current" (my wording) and "existing" (your wording) that the docs should clarify?